process_pr.py

```diff
@@ -2280,7 +2280,6 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
         x
         for x in labels
         if not x.endswith("-approved")
-        and not x.startswith("orp")
```

Comment:
@iarspider , isn't `and (not x.startwith("orp") or not issue.pull_request)` enough for this fix?

Reply:
Technically yes. I can change L2292 to check for `not issue.pull_request` instead of checking `signing_categories`, but would prefer to keep that condition separate, since it's more readable I think.